chanzuckerberg/czgenepi
# gisaid id clarification

### Summary
- **What:** Add clarity around what a `Public ID` is
- **Ticket:** [[sc-171823]](https://app.shortcut.com/genepi/story/171823)
- **Env:** https://[url]-frontend.dev.genepi.czi.technology
- **Design:** https://www.figma.com/proto/v5QPpEEbdwRmyxOzCXhrx7

### Testing
1. Verify tooltip appears when you hover over table header for `Public ID` column
1. Verify header text changed in metadata upload table
2. Verify header text changed in metadata template file

### Demos
<img width="1904" alt="Screen Shot 2021-11-17 at 1 36 32 PM" src="https://user-images.githubusercontent.com/7562933/142286636-8ad72912-5d5f-4cc7-a487-8cd0fd7f6469.png">
<img width="1008" alt="Screen Shot 2021-11-17 at 1 37 32 PM" src="https://user-images.githubusercontent.com/7562933/142286649-5fd6b48f-54fe-4c50-929b-14bf6aba1f87.png">
<img width="550" alt="Screen Shot 2021-11-17 at 1 38 07 PM" src="https://user-images.githubusercontent.com/7562933/142286653-cad1d6e2-2229-49e9-bf6a-144c60117fae.png">

### Checklist
